doc/monitoring/metrics_reference.rst

```diff
@@ -708,3 +708,172 @@ Event loop tx thread information:
             -   Event loop prolog time (ms)
         *   -   ``tnt_ev_loop_epilog_time``
             -   Event loop epilog time (ms)
+
+Memtx
+-----
+
+Memtx mvcc memory statistics.
+Transaction manager consists of two parts:
+- the transactions themselves (TXN section)
+- MVCC
+
+..  _metrics-reference-memtx_txn:
+
+TXN
+~~~
+
+``statements`` are the transaction statements.
+For example, the user started a transaction and made an action in it `space:replace{0, 1}`.
+Under the hood, this operation will turn into ``statement`` for the current transaction.
+
+..  container:: table
+
+    ..  list-table::
+        :widths: 25 75
+        :header-rows: 0
+
+        *   -   ``tnt_memtx_tx_tnx_statements_total``
+            -   The number of bytes that are allocated for the statements of all transactions.
+        *   -   ``tnt_memtx_tx_tnx_statements_avg``
+            -   Average bytes used by transactions for statements (`txn.statements.total` / number of open transactions).
+        *   -   ``tnt_memtx_tx_tnx_statements_max``
+            -   Maximum number of bytes used by one transaction for statements.
+
+``user``
+In Tarantool C API there are function `box_txn_alloc()`.
+By using this function user can allocate memory for the current transaction.
+
+..  container:: table
+
+    ..  list-table::
+        :widths: 25 75
+        :header-rows: 0
+
+        *   -   ``tnt_memtx_tx_tnx_user_total``
+            -   Memory allocated by the `box_txn_alloc()` function on all transactions.
+        *   -   ``tnt_memtx_tx_tnx_user_avg``
+            -   Transaction average (total / number of transactions).
+        *   -   ``tnt_memtx_tx_tnx_user_max``
+            -   The maximum number of bytes allocated by `box_txn_alloc()` function per transaction.
+
+``system`` - internals: logs, savepoints
+
+..  container:: table
+
+    ..  list-table::
+        :widths: 25 75
+        :header-rows: 0
+
+        *   -   ``tnt_memtx_tx_tnx_system_total``
```

Comment:
Same questions like in statement group about current transactions vs overall transactions.